calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,234 @@
+
+Note that the user should put the radar reflector out of the calibration area before clicking the button. Ideally, there shouldn't be any moving objects like humans and vehicles in the calibration area.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz1.jpg" alt="rviz1" width="500">
+</p>
+
+Once the user clicks the button, it will show like the image below.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz2.jpg" alt="rviz2" width="500">
+</p>
+
+Once the background is extracted, it will show like the image below. The user can see that the `Add lidar-radar pair` button is enabled.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz3.jpg" alt="rviz3" width="500">
+</p>
+
+Also, the following text should be shown in the console.
+
+```text
+[marker_radar_lidar_calibrator]: Finished background model initialization
+```
+
+Additionally, the user can click the options: `lidar_background_pointcloud`, `lidar_foreground_pointcloud`, and `radar_background_pointcloud` on the left bar of the `rviz` for visualizing the output of background and foreground pointcloud. As described in the [documentation](../../README.md), the `radar_reflections` is the same as the `radar_foreground_pointcloud`, which is visible as default.
+
+<table>
+  <tr>
+    <td><img src="../images/marker_radar_lidar_calibrator/lidar_background.jpg" alt="lidar_background" width = 700px height = 300px ></td>
+    <td><img src="../images/marker_radar_lidar_calibrator/lidar_foreground.jpg" alt="lidar_foreground" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Lidar background points.</p></td>
+    <td><p style="text-align: center;">Lidar foreground points.</p></td>
+  </tr>
+</table>
+
+<table>
+  <tr>
+    <td><img src="../images/marker_radar_lidar_calibrator/radar_background.jpg" alt="radar_background" width = 700px height = 300px></td>
+    <td><img src="../images/marker_radar_lidar_calibrator/radar_foreground.jpg" alt="radar_foreground" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Radar background points.</p></td>
+    <td><p style="text-align: center;">Radar foreground points.</p></td>
+  </tr>
+</table>
+
+### Add lidar-radar pair
+
+After the background model has been extracted, the user can carry the radar reflector (with the tripod) and place it in front of the radar sensor. Once the reflector is positioned, the user should step a few steps away from it. In the tutorial rosbag, the user will see that both the human and the radar reflector (with tripod) are identified as foreground objects in the image below.
+
+In the image, the green points represent the lidar foreground points, which form a cluster, while the purple points indicate radar foreground detections. The blue point is the estimated center of the radar reflector derived from the lidar pointcloud.
```

Comment:
My (perceived) color resolution is not very good, but both the reflector and human clusters can be called greeen.
I would say colored points represent clusters (different color = different cluster), and explain why humans do not make radar detections since we have such a good example

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/c21866ed48668c6d0a926fb0fb6940f210cee212

Reply:
Please avoid using the likes of "didnt" in technical writing
https://www.scienceeditingexperts.com/blog/use-of-apostrophes-in-scientific-writing

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/eea8ad07fa7a0fb024db88da60d18fa79273ce59

Reply:
Your fix has several grammatical errors. I think that after so many correction we do not should be dealing with these...

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/50a0fdddd7ce9af5f35a07e27ea9b9cdeb991e95